lib/rules/default-case.js

```diff
@@ -12,6 +12,8 @@ const DEFAULT_COMMENT_PATTERN = /^no default$/i;
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I'd consider `default-case` to be a `problem`, since a missing default is often a bug.

Reply:
I can go either way on this, let's see what others think.

Reply:
I feel similarly to @nzakas on this - I really feel like it could go either way.

Reply:
We currently have this listed under "Best Practices", so I think it makes sense to stick with "suggestion" for now.